gemrb/core/Interface.cpp

```diff
@@ -3422,9 +3437,12 @@ bool Interface::ReadItemTable(const ResRef& TableName, const char *Prefix)
 bool Interface::ReadRandomItems()
 {
 	ieDword difflev=0; //rt norm or rt fury
-	vars->Lookup("Nightmare Mode", difflev);
+	auto lookup = vars.find("Nightmare Mode");
+	if (lookup != vars.cend()) {
+		difflev = lookup->second;
+	}
```

Comment:
How about something like `ieDword difflev = GetVariable("Nightmare Mode", 0)` instead?

Reply:
Nothing, probably just an oversight since _GetVariable_ was introduced half-way after writing enough of this code :)